mmv1/third_party/terraform/services/vertexai/iam_vertex_endpoint_test.go.tmpl

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 func TestAccVertexAIEndpointIamBinding(t *testing.T) {
-	t.Parallel()
+	// t.Parallel()
```

Comment:
```suggestion
	// t.Parallel()
	// See https://github.com/hashicorp/terraform-provider-google/issues/18932
```

We could link the issue here to explain why these tests shouldn't be run in parallel.